packages/components/modules/profiles/web/ProfileComponent/index.tsx

```diff
@@ -102,6 +103,24 @@ const ProfileComponent: FC<ProfileComponentProps> = ({ profile: profileRef, curr
     )
   }
 
+  const menuOptions = (
+    <>
+      <MenuItem onClick={handleShareClick} disableRipple>
+        <ShareIcon />
+        Share profile
+      </MenuItem>
+      {profile && (
+        <BlockButtonWithDialog
+          target={profile}
+          handleCloseMenu={handleClose}
+          currentProfileId={currentProfileId}
+          isMenu
+        />
+      )}
+      {profile && <ReportButtonWithDialog handleClose={handleClose} targetId={profile?.id} />}
```

Comment:
why not add this on the prior block of {profile && ?

Reply:
Fixed